pages/blog/index.tsx

```diff
@@ -1,23 +1,70 @@
 import Link from "next/link";
 import React from "react";
-import styled from "styled-components";
 import { getBlogsMetadata } from "../../scripts/getBlogPosts";
 import Image from "../../components/Image/Image";
+import Adapt from "@components/UiKit/Adapt";
+import device from "@styles/utils/breakpoints";
+import styled from "styled-components";
+import { Heading3, BodyText } from "@components/UiKit/Typography";
 
-const Wrapper = styled.div`
-  padding: 100px 38px;
+const BlogWrapper = styled.div`
+  
 `;
+const BlogList = styled.ul`
+  display: inline-grid;
+  grid-template-columns: repeat(3, 1fr);
+  column-gap: 70px;
+  border-top: 1px solid ${({ theme }) => theme.borderColor};
+
+  @media ${device.L}{
+    column-gap: 35px;
+  }
 
-const Item = styled.li`
-  padding: 38px 0;
+  @media ${device.M}{
+    grid-template-columns: repeat(2, 1fr);
+  }
 
-  & + & {
-    border-top: 1px solid #ccc;
+  @media ${device.XS}{
+    grid-template-columns: repeat(1, 1fr);
   }
 `;
 
-const ThumbImageWrapper = styled.div`
-  max-width: 220px;
+const BlogListItem = styled.li`
+  padding-top: 30px;
+  padding-bottom: 40px;
+
+  &:nth-child(n+4) {
+    border-top: 1px solid ${({ theme }) => theme.borderColor};
+  }
+
+  @media ${device.M}{
+    &:nth-child(n+3) {
+      border-top: 1px solid ${({ theme }) => theme.borderColor};
+    }
+  }
+  @media ${device.XS}{
+    &:nth-child(n+2) {
+      border-top: 1px solid ${({ theme }) => theme.borderColor};
+    }
+  }
+`;
+
+const BlogListImageWrapper = styled.figure`
+  margin-bottom: 20px;
+
+  img {
+    aspect-ratio: 1.79;
+    object-fit: cover;
+    width: 100%;
+  }
+`;
+
+const BlogTag =  styled.p`
+  font-size: 1rem;
+  line-height: 1.25;
+  font-weight: ${({theme})=> theme.fontWeight600};
```

Comment:
Isn't it easier to just write `font-weight: 600`?

I don't see any benefit of havingh
```
  fontWeight400: 400,
  fontWeight600: 600,
  fontWright700: 700,
```
as a variable